aten/src/ATen/Context.cpp

```diff
@@ -59,6 +59,48 @@ void Context::setDeterministicCuDNN(bool b) {
   deterministic_cudnn = b;
 }
 
+bool Context::deterministic() const {
+  return _deterministic;
+}
+
+void Context::setDeterministic(bool b) {
+  _deterministic = b;
+}
+
+Context::ErrorLevel Context::deterministicErrorLevel() const {
+  return deterministic_error_level;
+}
+
+void Context::setDeterministicErrorLevel(Context::ErrorLevel e) {
+  deterministic_error_level = e;
+}
+
+Context::ErrorLevel Context::longToErrorLevel(long e) {
```

Comment:
Not sure why this is a method on Context, it has nothing to do with the Context...